iteration/iteration.go

```diff
@@ -39,6 +41,7 @@ type Iteration struct {
 	Name        string
 	Description *string
 	State       string // this tells if iteration is currently running or not
+	UserActive  *bool
```

Comment:
@DhritiShikhar do we still need this as a `pointer`? I understand that field is optional in API request but it is not optional in DB.

Reply:
Yes. We need that as a pointer because as you said that the field is optional in API request.
If in create request payload, user_active is not present, then i put user_active: false
If in update request payload, user_active is not present, then i use the value in db

Reply:
@DhritiShikhar to me your explanation doesn't explain why we need this as a pointer. To me it sounds like `nil` is the same as `false` in the way it can be handled. See my other comment above. IMHO you should at least be able to handle `UserActive == nil` because then you can remove a lot of the code above where you just set `UserActive` to a pointer of `false`.